docs/plugin-options.md

```diff
@@ -167,6 +191,14 @@ When true, all internal GraphQL queries generated during node sourcing will be w
 },
 ```
 
+## debug.timeBuildSteps
+
+_TODO_
+
+## debug.disableCompatibilityCheck
```

Comment:
@sslotsky on the call I mentioned taking these out, but I think let's leave them in like you have them as todo's. I'll make sure to mark some of them as internal options that could be removed at any time in the docs here. 🤔 maybe I should go through and prefix the options with ___internalOptionName in the code too or something.. This disableCompatibilityCheck one is one people shouldn't be using, but it's useful for testing the source plugin with new unsupported WPGraphQL versions. Technically anyone could find it in the source code and use it, but my thinking before was if it's undocumented, we can break it at any time 🤔